apps/root/src/common/components/wallet-selector/index.tsx

```diff
@@ -223,6 +223,20 @@ const WalletSelector = ({ options, size = 'small' }: WalletSelectorProps) => {
     type: OptionsMenuOptionType.option,
   };
 
+  const reconnectWalletOption: OptionsMenuOption = {
```

Comment:
I think this should be an option of the button and not the wallet selector, plus, does this actually fix the issue of the wallet appearing as '...'?